core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -496,6 +497,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
                     return Service.START_NOT_STICKY;
                 }
                 mediaPlayer.playMediaObject(playable, stream, startWhenPrepared, prepareImmediately);
+                skipIntro(playable);
```

Comment:
I don't think that this is enough. If AntennaPod automatically plays the next item in the queue, this code is not executed.